packages/browser-destinations/src/destinations/friendbuy/index.ts

```diff
@@ -0,0 +1,85 @@
+import type { Settings } from './generated-types'
+import type { BrowserDestinationDefinition } from '../../lib/browser-destinations'
+import { browserDestination } from '../../runtime/shim'
+import { defaultValues, DestinationDefinition } from '@segment/actions-core'
+
+import trackCustomer, { trackCustomerDefaultSubscription, trackCustomerFields } from './trackCustomer'
+import trackPurchase, { trackPurchaseDefaultSubscription, trackPurchaseFields } from './trackPurchase'
+import trackSignUp, { trackSignUpDefaultSubscription, trackSignUpFields } from './trackSignUp'
+import trackCustomEvent from './trackCustomEvent'
+
+export interface FriendbuyAPI extends Array<unknown> {
+  merchantId: string
+}
+
+declare global {
+  interface Window {
+    friendbuyAPI?: FriendbuyAPI
+    friendbuyBaseHost?: string
+  }
+}
+
+// Presets are shown in Segment configuration flow as "Pre-Built Subscriptions".
+const presets: DestinationDefinition['presets'] = [
+  {
+    name: 'Track Customer',
+    subscribe: trackCustomerDefaultSubscription,
+    partnerAction: 'trackCustomer',
+    mapping: defaultValues(trackCustomerFields)
+  },
+  {
+    name: 'Track Purchase',
+    subscribe: trackPurchaseDefaultSubscription,
+    partnerAction: 'trackPurchase',
+    mapping: defaultValues(trackPurchaseFields)
+  },
+  {
+    name: 'Track Sign Up',
+    subscribe: trackSignUpDefaultSubscription,
+    partnerAction: 'trackSignUp',
+    mapping: defaultValues(trackSignUpFields)
+  }
+]
+
+export const destination: BrowserDestinationDefinition<Settings, FriendbuyAPI> = {
+  name: 'Friendbuy',
+  slug: 'friendbuy',
```

Comment:
We already have a destination called `Friendbuy` so we're gonna need to change this.
We've been following this pattern:

```suggestion
  name: 'Friendbuy (Actions)',
  slug: 'actions-friendbuy',
```

How does it sound?